apps/skilavottord/ws/src/app/modules/recyclingPartner/recyclingPartner.resolver.ts

```diff
@@ -20,62 +28,38 @@ export class RecyclingPartnerResolver {
   async skilavottordAllActiveRecyclingPartners(): Promise<
     RecyclingPartnerModel[]
   > {
-    return await this.recyclingPartnerService.findActive()
+    return this.recyclingPartnerService.findAllActive()
```

Comment:
Not sure if this is correct ... this will cause errors for e.g.
1. create recycling partner with id 1337
2. make recycling partner <1337> inactive
3. now the recycling partner isn't shown in the UI, and can't thus be made active again. In addition to that, you can't create the same recycling partner since the ID of that partner has already been taken.

Reply:
There are 2 methods
`skilavottordAllRecyclingPartners` - returns all without any condition
`skilavottordAllActiveRecyclingPartners` - returns all active
 I only refactored the latter, but it is used in couple other places. Do u want me to replace every occurance with `skilavottordAllRecyclingPartners`?